packages/cli/src/util/print-help-information.ts

```diff
@@ -3,17 +3,13 @@ import type { BaseContext } from "clipanion";
 
 export function printHelpInformation(context: BaseContext) {
   context.stdout.write(
-    pc.black(
-      pc.red(
-        "If you need assistance, connect with us on our discord server: https://discord.gg/MJQ3WHktAS\n"
-      )
+    pc.red(
+      "\nIf you need assistance, connect with us on our discord server: https://discord.gg/MJQ3WHktAS\n"
```

Comment:
Probably a good place to also have a "Schedule office hours with a Panfactum engineer"

Reply:
@fullykubed is this a link, something to add to the website or just additional copy about what's available on Discord?